### PR TITLE
(bug) O3-4055 :  Add link to related issue 

### PR DESCRIPTION
**Pull request by jabahum**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

https://github.com/user-attachments/assets/49d76d51-625b-4464-87d6-9df74c09480e

image

## Related Issue

https://openmrs.atlassian.net/browse/O3-4055

### CODE REVIEW

**Comment by makombe** [comment]
@jabahum can you provide a screenshot/ video/description to get a glimpse of what the link does

File: src/stock-operations/add-stock-operation/add-stock-operation.component.tsx [hunk 1]
```diff
@@ -200,7 +201,7 @@ const AddStockOperation: React.FC<AddStockOperationProps> = (props) => {
         >
           <div style={{ margin: '10px' }}>
             {isEditing && (
-              <div style={{ display: 'flex', flexDirection: 'row' }}>
+              <div style={{ display: 'flex', flexDirection: 'row', marginLeft: '10px' }}>
```
Comment (makombe) [inline src/stock-operations/add-stock-operation/add-stock-operation.component.tsx:204]: @jabahum what do you think should we not manage css in it's own file for ease of maintainability?

File: src/stock-operations/add-stock-operation/stock-operation-status.component.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            <span className={styles.textHeading}>{t('completed', 'Completed')}:</span>
+            <div className={styles.statusDescriptions}>
+              <span className={styles.text}>
+                {formatDate(parseDate(props.model?.completedDate.toString()), {
+                  time: true,
+                  mode: 'standard',
+                })}
+              </span>
+
+              <span className={styles.text}>By</span>
```
Comment (makombe) [inline src/stock-operations/add-stock-operation/stock-operation-status.component.tsx:112]: let's translate `By`

File: src/stock-operations/add-stock-operation/stock-operation-status.component.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+            <span className={styles.textHeading}>{t('rejected', 'Rejected')}:</span>
+            <div className={styles.statusDescriptions}>
+              <span className={styles.text}>
+                {formatDate(parseDate(props.model?.rejectedDate.toString()), {
+                  time: true,
+                  mode: 'standard',
+                })}
+              </span>
+
+              <span className={styles.text}>By</span>
```
Comment (makombe) [inline src/stock-operations/add-stock-operation/stock-operation-status.component.tsx:155]: Same here

File: src/stock-operations/add-stock-operation/stock-operation-status.component.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+            <span className={styles.textHeading}>{t('submitted', 'Submitted')}:</span>
+            <div className={styles.statusDescriptions}>
+              <span className={styles.text}>
+                {formatDate(parseDate(props.model?.submittedDate.toString()), {
+                  time: true,
+                  mode: 'standard',
+                })}
+              </span>
+
+              <span className={styles.text}>By</span>
```
Comment (makombe) [inline src/stock-operations/add-stock-operation/stock-operation-status.component.tsx:48]: Make this translatable.

**Comment by makombe** [comment]
@jabahum it will also be good to have a ticket for this for future reference.  Thanks

**Comment by jabahum** [comment]
> @jabahum it will also be good to have a ticket for this for future reference. Thanks

i have updated the ticket name

**Review by ODORA0 — APPROVED** [review]

**Pull request merged**